packages/datadog-plugin-http/src/client.js

```diff
@@ -158,7 +158,17 @@ function patch (http, methodName, tracer, config) {
   }
 
   function normalizeArgs (inputURL, inputOptions, callback) {
-    let options = typeof inputURL === 'string' ? url.parse(inputURL) : Object.assign({}, inputURL)
+    let options
```

Comment:
It would be cleaner to put the new code in a function that returns the options (including the existing `options.headers` line below). Since it handles only the URL and not the actual options, maybe something like `extractURL`?.

Reply:
sounds good, updated this. Fwiw, there's already a similar `extractUrl` method name which could get confusing so how about `normalizeUrl` ?